templates/profile/prometheus/rules.yml.erb

```diff
@@ -160,6 +160,33 @@ groups:
     annotations:
       summary: 'Dark Blue repository storage, {{$labels.device}} is out of available space.'
       description: '{{$labels.device}} has had less than 100GB of available space for more than 5 minutes.'
+  - alert: MysqlDown
+    expr: >
+      mysql_up == 0
+    for: 5m
+    labels:
+      severity: page
+    annotations:
+      summary: 'MySQL is DOWN on {{$lables.hostname}}'
+      description: 'MySQL is DOWN on {{$labels.hostname}}'
+  - alert: MysqlSlaveSqlDown
+    expr: >
+      mysql_slave_status_slave_sql_running  == 0
+    for: 5m
+    lablels:
+      severity: page
+    annotations:
+      summary: 'MySQL Slave SQL is Not Running'
+      description: 'MySQL Slave SQL is not Running on {{$labels.hostname}}'
+  - alert: MysqlMaxPreparedStatment
+    expr: >
+      mysql_global_status_prepared_stmt_count / mysql_global_variables_max_prepared_stmt_count > 0.75
+    for: 30m
+    lables:
+      severity: ticket
+    annotations:
+      summary: 'MySQL Prepared Statment Count is apparoaching MAX on {{$lables.hostname}}'
+      description: 'MySQL Prepared Statment Count is approaching MAX on {{$labels.hostname}}'
```

Comment:
typos

Reply:
Brain and fingers not in agreement when I wrote this one.  fixed.